app/config/ViewConfig.scala

```diff
@@ -42,6 +44,7 @@ case class ViewConfig(
   val signOut = s"$frontendBaseUrl/vat-repayment-tracker/signout"
   val showResultsUrl = s"$frontendBaseUrl/vat-repayment-tracker/show-vrt"
   val nonMtdUser = s"$frontendBaseUrl/vat-repayment-tracker/non-mtd-user"
+  def vatVariationsUrl(vrn: Vrn) = s"https://www.tax.service.gov.uk/vat-variations/org/${vrn.value}/introduction"
```

Comment:
Are we sure that we don't need to vary this based on the environment where we run the app?
Jake will be thankful for that ;)

It should be fairly easy to define a base url in config (to be overriden on dev/qa/etc) and rephrase into:
```
 val appBaseurl: String = ..
 def vatVariationsUrl(vrn: Vrn) = s"$ appBaseurl/vat-variations/org/${vrn.value}/introduction"

```

Reply:
I asked Alfie this ... we agrees to leave static.  I can do the above though as a compromise